pythia/models/top_down_bottom_up.py

```diff
@@ -15,7 +15,7 @@
 @registry.register_model("top_down_bottom_up")
 class TopDownBottomUp(BaseModel):
     def __init__(self, image_attention_model, text_embedding_models, classifier):
-        super(TopDownBottomUpModel, self).__init__()
+        super(TopDownBottomUp, self).__init__()
```

Comment:
Can you actually change it to `super().__init__()`? Since, we are not supporting Pythia 2.x, we should do it in 3.x's way.

Reply:
Done :)

Reply:
Can you sign the CLA as well? Thanks.

Reply:
I actually did! Not showing though

Reply:
Thanks.